src/extension/app/components/action-bar/picker/picker.css.js

```diff
@@ -150,12 +150,41 @@ export const style = css`
     background-color: var(--spectrum-gray-400);
   }
 
+  :host([chevron="false"]) #button sp-icon-chevron100 {
+    display: none;
+  }
+
+  :host([quiet]) #button {
+    margin-top: 0;
+  }
+
+  :host([quiet]) #button:hover {
+    background-color: var(--spectrum2-edit-background-hover);
+  }
+
+  :host([quiet]) #button #label {
+    margin-top: 0;
+    margin-block-end: 0;
+    padding: 0 12px;
+    font-size: var(--spectrum-global-dimension-font-size-100);
+  }
+
+  :host([quiet]) #button .picker {
+    margin-inline-start: 0;
+    margin-inline-end: 8px;
+  }
+
   sp-menu {
     padding: 12px;
     gap: 4px;
   }
 
   sp-overlay sp-popover {
     backdrop-filter: var(--spectrum2-sidekick-backdrop-filter);
+    padding: 0;
+  }
+
+  sp-tray {
```

Comment:
Not sure I understand where `sp-tray` comes into play?

Reply:
It is used by spectrum to display popovers on mobile.

Reply:
I'll remove this rule and leave the default position for now. This will need to me sorted out for other popovers as well.